docs/tutorial.rst

```diff
@@ -454,5 +454,46 @@ To address this issue, DREST fields like DynamicField and DynamicMethodField sup
                 user.profile.preferred_last_name
             )
 
+Following relations using a slug instead of an id
+-------------------------------------------------
 
+DREST auto-generates an endpoint to return just the related data (useful as link objects). By default
```

Comment:
Does this feature only impact related/link endpoints, or would it apply to the detail endpoint as well? In other words, would `localhost:9002/events/new-event/` work as well as `localhost:9002/events/5/`?

 It seems like the latter, but this documentation suggests it is specifically for links.